### PR TITLE
Add horizontal margin to form preview when template is not found

### PR DESCRIPTION
**Pull request by AbdiTolesa**
Fix https://github.com/Strategy11/formidable-pro/issues/4223

image

### CODE REVIEW

**Review by garretlaxton — APPROVED** [review]
This looks good!

File: css/custom_theme.css.php [hunk 1]
```diff
@@ -110,6 +110,10 @@
 <?php } ?>
 }
 
+.frm-preview {
```
Comment (Crabcyborg) [inline css/custom_theme.css.php:113]: @AbdiTolesa We don't want to add any special preview code into our front end CSS. We want the CSS to be as small as possible.

Comment (Crabcyborg) [reply css/custom_theme.css.php:113]: I also don't love that this is always applied. This update should really only change how a block template is rendered.

File: classes/controllers/FrmFormsController.php [hunk 2]
```diff
@@ -397,6 +397,7 @@ private static function load_theme_preview() {
 		add_filter( 'is_active_sidebar', '__return_false' );
 		FrmStylesController::enqueue_css( 'enqueue', true );
 
+		$p = get_query_template( 'page' );
```
Comment (Crabcyborg) [inline classes/controllers/FrmFormsController.php:400]: @AbdiTolesa What is up with this line? `$p` is never used.

**Comment by shervElmi** [comment]
@AbdiTolesa, I'm relatively new to the FF team. Could you please provide the steps to reproduce the issue in this PR?
Detailed steps would be incredibly helpful and time-saving for me. Thank you!

**Comment by Crabcyborg** [comment]
I'm going to close this for now as I don't think we'll re-use any of these updates in the final solution.

**Pull request closed without merging**